docs/content/tutorial/resize-cluster.md

```diff
@@ -15,9 +15,19 @@ Let's dive in.
 
 Memory and CPU resources are an important component for vertically scaling your Postgres cluster. Couple with [tweaks to your Postgres configuration file]({{< relref "./customize-cluster.md" >}}), allowing your cluster to have more memory and CPU allotted to it can help it to perform better under load.
 
-It's important for instances in the same high availability set to have the same resources. PGO  lets you adjust CPU and memory within the `spec.instances.resources` section of the `postgresclusters.postgres-operator.crunchydata.com` custom resource. The layout of `spec.instances.resources` should be familiar: it follows the same pattern as the standard Kubernetes structure for setting [container resources](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/).
+It's important for instances in the same high availability set to have the same resources. PGO lets you adjust CPU and memory within the `resources` sections of the `postgresclusters.postgres-operator.crunchydata.com` custom resource. These include:
```

Comment:
Weren't we going to put these new docs in the `Customize-cluster.md` docs :question:

Reply:
That was the original idea, but on review we already use had a resource configuration example in `resize-cluster.md`, so it didn't make sense to me to add it again in another section.